community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/StoreUpgrader.java

```diff
@@ -318,11 +319,12 @@ public UpgradingStoreVersionNotFoundException( String filenameWithoutStoreVersio
 
     public static class UnexpectedUpgradingStoreVersionException extends UnableToUpgradeException
     {
-        protected static final String MESSAGE = "'%s' has a store version '%s' that we cannot upgrade from.";
+        protected static final String MESSAGE =
+                "Unable to upgrade. Cannot open store with version '%s'. Current version is `%s` (Neo4j %s).";
```

Comment:
I find this message a bit unclear: we say that current version is _something_and then mention neo4j version. What was that something and what i can do with it to be able to fix?